### DISCUSSION TITLE
Feature Request: Add "New Update Set" icon in the Update Set switcher

### DISCUSSION THREAD

**Discussion by tdeniston** [arnoudkooi/SN-Utils#270] [Ideas]
Having gotten used to the UI16 Developer Patch's feature where the Update Set icon turns into a menu (first screenshot), I really miss being able to quickly create a new update set. I'd like to propose having SN Utils add an icon to the Update Set switcher in Next UI that would direct the user to a new update set record. The icon would fit nicely next to the existing "Open list" and "Open record" icons (second screenshot). 

<img width="221" alt="image" src="https://user-images.githubusercontent.com/15730875/180480883-c19637bf-d282-41d8-bed9-b9819b2fc697.png">

<img width="429" alt="image" src="https://user-images.githubusercontent.com/15730875/180480664-64118410-0b14-470c-b688-846839101310.png">

**Comment by arnoudkooi** [comment]
This menu is only in the dom when it is open, so it is hard to hook into, 
maybe a custom slashcommand is an option? `sys_update_set.do?sys_id=-1`
